testsmanual/BenchmarkDotNet.IntegrationTests.ClassicNetFramework/ClassicNetFrameworkTest.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Tests.XUnit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace BenchmarkDotNet.IntegrationTests
+{
+    public class ClassicNetFrameworkTest : BenchmarkTestExecutor
+    {
+        private const string Prefix = "// ### Called: ";
+        private const string GlobalSetupCalled = Prefix + "GlobalSetup";
+        private const string GlobalCleanupCalled = Prefix + "GlobalCleanup";
+        private const string IterationSetupCalled = Prefix + "IterationSetup";
+        private const string IterationCleanupCalled = Prefix + "IterationCleanup";
+        private const string BenchmarkCalled = Prefix + "Benchmark";
+
+        private readonly string[] expectedLogLines = {
+            "// ### Called: GlobalSetup",
+
+            "// ### Called: IterationSetup (1)", // MainWarmup1
+            "// ### Called: Benchmark", // MainWarmup1
+            "// ### Called: IterationCleanup (1)", // MainWarmup1
+            "// ### Called: IterationSetup (2)", // MainWarmup2
+            "// ### Called: Benchmark", // MainWarmup2
+            "// ### Called: IterationCleanup (2)", // MainWarmup2
+
+            "// ### Called: IterationSetup (3)", // MainTarget1
+            "// ### Called: Benchmark", // MainTarget1
+            "// ### Called: IterationCleanup (3)", // MainTarget1
+            "// ### Called: IterationSetup (4)", // MainTarget2
+            "// ### Called: Benchmark", // MainTarget2
+            "// ### Called: IterationCleanup (4)", // MainTarget2
+            "// ### Called: IterationSetup (5)", // MainTarget3
+            "// ### Called: Benchmark", // MainTarget3
+            "// ### Called: IterationCleanup (5)", // MainTarget3
+
+            "// ### Called: GlobalCleanup"
```

Comment:
the tests that verify the order of printed output are flaky. If all we care about is ensuring that .NET Framework works, we should just verify the runtime in the benchmark and throw if it's the wrong one